go.mod

```diff
@@ -202,7 +202,7 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/gocql/gocql => github.com/scylladb/gocql v1.14.4
+replace github.com/gocql/gocql => github.com/scylladb/gocql v1.15.0
```

Comment:
why are we moving to the new version ? what in it ?

please do this on a separated PR, and with proper commit message

we should be doing too many changes at once, we got bitten with it too many times now...

Reply:
This is already in master, something during merge made it into here